README.md

```diff
@@ -189,16 +191,16 @@ be read from standard input, e.g.:
     thoth: using database at thoth-db:28015 (configured from Manta)
     thoth: reading dump identifiers from stdin
     NAME             TYPE  TIME                NODE/CMD         TICKET
-    3260a5e49918260c core  2017-11-16T22:30:22 pg_prefaulter    -     
-    5aaa91149e94a91f core  2017-11-17T11:07:43 pg_prefaulter    -     
-    04a681f27ffcd199 core  2017-11-17T14:12:27 pg_prefaulter    -     
-    3f7a8bde5a907afa core  2017-11-17T14:42:03 pg_prefaulter    -     
-    f12ea8712e8b2586 core  2017-11-17T17:22:21 pg_prefaulter    -     
+    3260a5e49918260c core  2017-11-16T22:30:22 pg_prefaulter    -
+    5aaa91149e94a91f core  2017-11-17T11:07:43 pg_prefaulter    -
+    04a681f27ffcd199 core  2017-11-17T14:12:27 pg_prefaulter    -
+    3f7a8bde5a907afa core  2017-11-17T14:42:03 pg_prefaulter    -
+    f12ea8712e8b2586 core  2017-11-17T17:22:21 pg_prefaulter    -
 
 ## Subcommands
 
 `thoth` operates by specifying a subcommand.  Many subcommands kick
-off Manta jobs, and the job ID is presented in the command line
+off Manta jobs when using v1, and the job ID is presented in the command line
```

Comment:
Is the intent to have this version of thoth support using jobs still if installed to a mantav1?

Reply:
I'll come back to the rest of your comments (thanks), but to answer this one early: yes. Specifically for 'analyze' and 'debug', it still runs on the storage node not locally.